jovian/cli.py

```diff
@@ -109,23 +110,21 @@ def exec_clone(ctx, notebook, version, no_outputs, overwrite):
 
     Or clone a specific version of notebook:
 
-        $ jovian clone aakashns/jovian-tutorial -v 10
```

Comment:
this can be reverted

Reply:
Click's usage shows [OPTIONS] first.
![image](https://user-images.githubusercontent.com/24864161/85282262-11d70b80-b4a9-11ea-89c5-f3930c3fc9c8.png)

Will it be add options after argument ?